### ISSUE TITLE
Unable to generate context files: metadata contain unresolvable link

### ISSUE THREAD

**Issue by Cherepoc** [delegateas/XrmContext#17]
Hello! Trying to use your code generator, but unfortunately can't get it to work. It always says Unable to generate context files: metadata contain unresolvable link (in my native language, not shure how this error reads in english). It also displays the url I give to it and it always adds ?wsdl&sdkversion=0. What can I do with that? Please ask if you need any other info. Thanks!
Btw CrmSvcUtil works fine with the same arguments.

**Comment by henrikhannemose** [comment]
Thanks for trying to use XrmContext. Which version of CRM are you connecting to?

**Comment by Cherepoc** [comment]
Microsoft Dynamics 365
Version 1612 (8.2.2.112) (DB 8.2.2.112)

**Comment by henrikhannemose** [comment]
I have tried connecting to an on-premises CRM having _version 1612 (8.2.2.112) (DB 8.2.2.112)_ with XrmContext 1.5.0. That sounds like the exact same version you are connecting to. For me it works fine, so I am unfortunately not able to reproduce the issue.

I have taken a general look at XrmContext, and could see that it was using v8 assemblies. Your issue reminds me of the general TLS 1.2-related issue from earlier in the year, so I have updated the assemblies to a newer version to see if that changes anything. Are you able to try with [version 1.5.1](https://www.nuget.org/packages/Delegate.XrmContext/1.5.1) to see how it behaves for you?
